labgrid/driver/ubootdriver.py

```diff
@@ -206,13 +206,13 @@ def await_boot(self):
         self.console.expect(self.bootstring, timeout=self.boot_timeout)
 
     @Driver.check_active
-    @step(args=['name'])
-    def boot(self, name):
-        """Boot the default or a specific boot entry
+    @step(args=['cmd'])
+    def boot(self, cmd: str = ""):
+        """Boot the default if cmd is empty or a specific boot cmd otherwise.
```

Comment:
The LinuxBootProtocol defines this parameter as `cmd`: https://github.com/labgrid-project/labgrid/blob/master/labgrid/protocol/linuxbootprotocol.py#L6

We could have a dict in the UBootDriver configuration which defines commands for each boot "entry". What do you think? @jremmet @Emantor

Reply:
I like the idea. I would be configurable, and we can check if exists. Guess `self.boot_command`should continue to work and the default for `name = ""` ? @jluebbe

Reply:
Yes. Perhaps something like:
```
boot_commands:
  A: "run boot_slot_a"
  B: "run boot_slot_b"
```

Reply:
This way, you could also reuse test cases and Strategies easier with different boards.